src/test/kotlin/Decorator/littlekai/base/IngredientDecoratorTest.kt

```diff
@@ -1,15 +1,54 @@
 package Decorator.littlekai.base
 
+import Decorator.littlekai.ingredients.Pork
+import Decorator.littlekai.ingredients.Tuna
+import Decorator.littlekai.noodles.EggNoodles
+import Decorator.littlekai.noodles.UdonNoodles
+import Decorator.littlekai.noodles.WheatNoodles
 import org.hamcrest.CoreMatchers.`is`
 import org.junit.Assert.assertThat
+import org.junit.Assert.assertTrue
 import org.junit.Test
 
 class IngredientDecoratorTest {
 
+  val baseNoodles = UdonNoodles()
+  val ingredient = Pork(baseNoodles)
+  val doubleIngredient = Tuna(ingredient)
+
+  @Test
+  fun `check if decorated element is still Noodles`() =
+    assertTrue(ingredient is Noodles)
+
+
+  @Test
+  fun `check type of stacked decorators should be equal to last`() =
```

Comment:
You are not tested your model, you are tested how you have created your objects in your tests.
You are not tested nothing related with your domain, only how are you created a test.

If the creation of objects for example were val doubleIngredient = Pork(noodle) your test would be like this: 

```
fun `check type of stacked decorators should be equal to last`() = assertTrue(double is Pork)
```